<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelMainImg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxNavigator">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Navigator:</string>
    </property>
    <widget class="QLabel" name="labelText_load_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Load Image:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_browse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelText_image_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Image Info:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelData_dimensions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string>Dimensions: WxH</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_segment">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Segment Image</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
